src/commands/azureServiceOperators/helpers/azureservicehelper.ts

```diff
@@ -64,47 +59,35 @@ async function getOperatorsPod(
     clusterKubeConfig: string
 ): Promise<k8s.KubectlV1.ShellResult | undefined> {
     // kubectl get pods -n operators
-    const command = `get pods -n operators`;
-    const failureDescription = "Get operator pod had following error";
+    const command = `get pods -n azureserviceoperator-system`;
+    const failureDescription = "Get azureserviceoperator-system pod had following error";
 
     const result = await invokeKubectlCommand(kubectl, clusterKubeConfig, command, failureDescription);
     return result;
 }
 
-async function installOlmCrd(
+async function installCerManager(
     kubectl: k8s.KubectlV1,
     clusterKubeConfig: string
 ): Promise<k8s.KubectlV1.ShellResult | undefined> {
     try {
-        const asoCrdYamlFile = "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.3/crds.yaml";
+        const asoCrdYamlFile = "https://github.com/jetstack/cert-manager/releases/download/v1.7.1/cert-manager.yaml";
         const runResult = await tmpfile.withOptionalTempFile<k8s.KubectlV1.ShellResult | undefined>(
             clusterKubeConfig, "yaml",
             (f) => kubectl.invokeCommand(`create -f ${asoCrdYamlFile} --kubeconfig="${f}"`));
 
         return runResult;
     } catch (e) {
-        vscode.window.showErrorMessage(`Installing operator lifecycle manager CRD resource had following error: ${e}`);
+        vscode.window.showErrorMessage(`Installing cert-manager resource had following error: ${e}`);
         return undefined;
     }
 }
 
-async function installOlm(
-    kubectl: k8s.KubectlV1,
-    clusterKubeConfig: string
-): Promise<k8s.KubectlV1.ShellResult | undefined> {
-    const asoOlmYamlFile = "https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.3/olm.yaml";
-    const command = `create -f ${asoOlmYamlFile}`;
-    const failureDescription = "Installing operator lifecycle manager resource had following error";
-
-    const result = await invokeKubectlCommand(kubectl, clusterKubeConfig, command, failureDescription);
-    return result;
-}
-
 async function installOperator(
     kubectl: k8s.KubectlV1,
     clusterKubeConfig: string
 ): Promise<k8s.KubectlV1.ShellResult | undefined> {
-    const asoYamlFile = "https://operatorhub.io/install/azure-service-operator.yaml";
+    const asoYamlFile = "https://github.com/Azure/azure-service-operator/releases/download/v2.0.0-beta.0/azureserviceoperator_v2.0.0-beta.0.yaml";
     const command = `create -f ${asoYamlFile}`;
```

Comment:
I know this was already like this, but could we use `apply` instead of `create` here? When I run the command, I get lots of errors because resources already exist (I previously installed an older ASO version). I _think_ those failures would be avoided by using `apply`...

Reply:
💡 We used create for the `error` reason, I guess if `ASO` is already installed, we want user to know that you either installed or underlying stuff already installed so take a look. That also takes out unnecessary  handling of if installed half way etc sceanrios.